src/main/java/org/jenkinsci/test/acceptance/docker/DockerImage.java

```diff
@@ -16,16 +19,26 @@
 
     public static final String DEFAULT_DOCKER_HOST = "127.0.0.1";
     public final String tag;
-    DockerHostResolver dockerHostResolver;
+    static DockerHostResolver dockerHostResolver = new DockerHostResolver();
 
     public DockerImage(String tag) {
         this.tag = tag;
-        dockerHostResolver = new DockerHostResolver();
     }
 
-    public <T extends DockerContainer> T start(Class<T> type, CommandBuilder options, CommandBuilder cmd,int portOffset) throws InterruptedException, IOException {
+    public <T extends DockerContainer> T start(Class<T> type, CommandBuilder options, CommandBuilder cmd, int portOffset) throws InterruptedException, IOException {
         DockerFixture f = type.getAnnotation(DockerFixture.class);
-        return start(type,f.ports(),portOffset,f.bindIp(),options,cmd);
+
+        String addr = f.bindIp();
```

Comment:
Didn't you deprecate `bindIp`? And wouldn't that mean that you are no longer using it?